tests/shared.py

```diff
@@ -336,7 +336,7 @@ def no_internet_connection():
         conn = socket.create_connection((host, 80), 2)
         conn.close()
         return False
-    except socket.gaierror:
+    except (socket.gaierror, TimeoutError):
```

Comment:
Unrelated, but this popped up while I was testing. It looked like a random network error